Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN apk update \
 
 COPY Gemfile* package.json yarn.lock ./
 COPY ./config/secrets.yml.example ./config/k8s/secrets.yml
+COPY ./config/secrets.yml.example ./config/secrets.yml
```

Comment:
Instead of this, add the following to the end of config/application.rb

    # Any standard rails config files placed in config/k8s will override the corresponding files
    # in config/. This allows us to use config/k8s as a secrets mount.
    if Dir.exist?("config/k8s")
      Dir.glob("config/k8s/*.{yml,key,yml.enc}") do |filename|
        paths.add "config/#{File.basename(filename).split('.')[0]}", with: filename
      end
    end

Reply:
Thanks! I made the change